cmd/main.go

```diff
@@ -22,6 +22,7 @@ import (
 	"sigs.k8s.io/kubebuilder/v3/pkg/cli"
 	cfgv2 "sigs.k8s.io/kubebuilder/v3/pkg/config/v2"
 	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
+	goconfiggenv1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang-config-gen/v1"
```

Comment:
Let's use alpha here as well `go-config-gen/alphav1` until it gets mature enough to be v1.

Reply:
The package name doesn't really matter IMO, since the plugin has a `v1-alpha` version.

Reply:
I think that matter because:
- make explicitly that is in alpha for maintainers and consumers
- kb is also a lib/api and we need to make it clear that we can break everything that we wish with this one.